packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/getLocationDetailViewTableData/constants.ts

```diff
@@ -4,7 +4,7 @@ export const LOCATION_DETAIL_VIEW_HEADERS: LocationDetailViewHeaders = [
   { key: 'checkbox', type: 'text', content: { text: '' } },
   { key: 'name', type: 'sort', content: { label: 'Name' } },
   { key: 'type', type: 'sort', content: { label: 'Type' } },
-  { key: 'last-modified', type: 'sort', content: { label: 'Last Modified' } },
+  { key: 'last-modified', type: 'sort', content: { label: 'Last modified' } },
```

Comment:
These should be using the display text API. Pretty certain the correct headers exist there but assuming this table was missed in the PRs integrating the api with other tables